corehq/apps/locations/static/locations/ko/location.js

```diff
@@ -0,0 +1,33 @@
+// for product and user per location selection
+$(function () {
+    $("#id_users-selected_ids").width(800).height(400).multiselect();
+    $("#id_products-selected_ids").width(800).height(400).multiselect();
+});
+
+var insert_new_user = function(user) {
+    $('#id_users-selected_ids').multiselect('addSelected', user.user_id, user.text);
+};
+
+$('#new_user').on('show', function() {
+
+    var url = NEW_MOBILE_WORKER_URL + "?location_id=" + LOCATION_ID;
+    var form_node = $("#new_user form");
+
+    $.get(url, function(data) {
```

Comment:
This modal can be called several times, and it contains a request/response workflow to the server, as it needs to be re-rendered with errors.  It could probably be preloaded initially, but I'm not sure that that has much of an advantage, as subsequent usages would need to hit the server anyways.